parsing-utilities/src/main/java/gov/nasa/jpl/aerie/json/package-info.java

```diff
@@ -101,7 +101,7 @@
  * which specifies a JSON object whose fields are described by other parsers -- rather than using {@code integerExprP}
  * directly. There are two reasons for this! First, we want parsers to be "productive", which means that they should
  * consume some part of the input before descending into a subparser. This is not always a hard-and-fast rule, but since
- * our grammar is recursive, we <i>>must</i make progress on the input before cycling back to the same point in the
+ * our grammar is recursive, we <i>>must</i> make progress on the input before cycling back to the same point in the
```

Comment:
Could there be any issues with the second `>` in `<i>>must`?

Reply:
Ye gads, so *that's* where it went!

Reply:
No, there shouldn't be any issues *technically* (especially since `./gradlew javadoc`) worked. An extra `>` outside of an individual tag can only be interpreted as a literal `>`.

But... it's obviously incorrect. Fixed :)